examples/tutorials/first-figure.py

```diff
@@ -80,7 +82,7 @@
 # 1. The name of method is ``coast`` instead of ``pscoast``. As a general rule, all
 #    ``ps*`` modules had their ``ps`` prefix removed. The exceptions are:
 #    ``psxy`` which is now ``plot``, ``psxyz`` which is now ``plot3d``, and ``psscale``
-#    which is now ``colorbar``.
+#    which is now ``colorbar``. Those reflect the new GMT modern execution mode.
```

Comment:
It seems that only point 1 reflects the differences between classic GMT and GMT modern mode, while points 2-5 are PyGMT exclusive. Shall we divide 1-5 into two parts and also update the introduction? One discusses the differences between classic GMT and GMT modern mode, and the other is about PyGMT.